solidity/contracts/bridge/Bridge.sol

```diff
@@ -1460,68 +1474,38 @@ contract Bridge is Ownable {
 
```

Comment:
We could add to point 10:

> The wallet is expected to finish the already queued redemption requests before moving funds but we are not going to enforce it on-chain.

Reply:
See https://github.com/keep-network/tbtc-v2/pull/128/commits/64395ffafef55a2795e927a1dad3c531d0ef24d8